src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -525,10 +525,12 @@ parseIdentifierAndBinder = do
 -- Parse a binder
 --
 parseBinder :: TokenParser Binder
-parseBinder = withSourceSpan PositionedBinder (P.buildExpressionParser operators parseBinderAtom)
+parseBinder = withSourceSpan PositionedBinder (P.buildExpressionParser operators (buildPostfixParser postfixTable parseBinderAtom))
   where
   -- TODO: remove this deprecation warning in 0.8
   operators = [ [ P.Infix (P.try $ C.indented *> colon *> featureWasRemoved "Cons binders are no longer supported. Consider using purescript-lists or purescript-sequences instead.") P.AssocRight ] ]
```

Comment:
I think we can probably remove this deprecation warning by now, what do you think @garyb?